ui/src/__tests__/spec/tests/services.spec.js

```diff
@@ -82,35 +84,28 @@ describe('services screen tests', () => {
         await modalDeleteButton.click();
     });
 
-    describe('services screen tests', () => {
-        it('when clicking "Allow" button on a provider without having appropriate authorisation, the error should be displayed to the right of the button', async () => {
-            // open browser
-            await browser.newUser();
-            await browser.url(`/`);
-
-            // select domain
-            let domain = 'athenz.dev.test-non-admin';
-            let testDomain = await $(`a*=${domain}`);
-            await browser.waitUntil(async () => await testDomain.isClickable());
-            await testDomain.click();
+    it('when clicking "Allow" button on a provider without having appropriate authorisation, the error should be displayed to the right of the button', async () => {
+        await console.log(`testtesttest inside second test`)
+        // open browser
+        await browser.newUser();
+        await browser.url(`/domain/athenz.dev.test-non-admin/role`);
```

Comment:
fix of the second test - navigate to domain directly via url, since the user is not its memeber and thus domain is not visible in the list of domains on the right in ui